app/jobs/sync_custom_filter_count_job.rb

```diff
@@ -0,0 +1,9 @@
+class SyncCustomFilterCountJob < ApplicationJob
+  queue_as :low
+
+  def perform(filter)
+    Redis::Alfred.set(filter.filter_count_key, 0) if filter.filter_records.nil?
```

Comment:
Do we need this explicitly, won't it be handled in `filter.set_record_count_in_redis`?

Reply:
@scmmishra  As in it will be the same, there we would need to add 0 for nil records values.